package.json

```diff
@@ -38,40 +33,45 @@
 		"phaser-ce": "2.16.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.9.0",
-		"@babel/plugin-transform-runtime": "^7.10.3",
-		"@babel/preset-env": "^7.9.5",
+		"@babel/core": "^7.20.7",
+		"@babel/plugin-transform-runtime": "^7.19.6",
+		"@babel/preset-env": "^7.20.2",
 		"@types/jquery": "^3.5.9",
-		"@typescript-eslint/eslint-plugin": "^5.6.0",
-		"@typescript-eslint/parser": "^5.6.0",
-		"babel-loader": "^8.1.0",
+		"@typescript-eslint/eslint-plugin": "^5.47.1",
+		"@typescript-eslint/parser": "^5.47.1",
+		"babel-loader": "^9.1.0",
 		"babel-plugin-transform-runtime": "^6.23.0",
 		"babel-polyfill": "^6.26.0",
-		"css-loader": "^3.5.2",
-		"eslint": "^8.4.1",
-		"eslint-config-prettier": "^8.3.0",
-		"eslint-plugin-prettier": "^4.0.0",
-		"expose-loader": "^0.7.5",
-		"file-loader": "^6.0.0",
-		"html-loader": "^1.1.0",
-		"html-webpack-plugin": "^4.2.0",
+		"copy-webpack-plugin": "^6.2.1",
+		"cross-env": "^7.0.2",
+		"css-loader": "^6.7.3",
+		"dotenv-defaults": "^5.0.2",
+		"dotenv-webpack": "^8.0.1",
+		"dotenv": "^16.0.3",
```

Comment:
The `dotenv` related dependencies do my head, especially since I have the flu for a while now.
So basically some default `.env.example` is provided, which contains for convenience some online multiplayer test server credentials (server is down atm and credentials might change soon anyway). Game deployed for production will not have devDeps installed and might use `.env` file properly configured (so that hopefully works well even without `dotenv` installed) or some environment variables entered from some UI , depending on host. Just a note for myself overall, need to make sense of things and figuring out if dotenv stuff move into devDeps makes sense :)

Reply:
`dotenv` can be a regular dependency or a dev dependency, depending on how it's used. If it's only used at build time like it is for us (webpack config), then it can be a dev dep and remove some size from the production bundle.

More info: https://github.com/motdotla/dotenv/issues/216

Reply:
@nandastone I'm aware, but as I was saying, we're not using it only for webpack, but for online multiplayer server configuration as well. So I'll have to keep in mind if to move those back at some point. Thanks for the link, always good to see examples. Anyway, online multiplayer been dropped from next release because one critical bug hasn't been fixed for over a year now, so will see about it again in v0.5's development, hence why not fiddling with `dotenv` stuff atm.

Reply:
Ok, happy to revert to regular dependency if needed - no big deal.